src/wallet/rpcwallet.cpp

```diff
@@ -5354,13 +5493,18 @@ UniValue z_setmigration(const UniValue& params, bool fHelp) {
 UniValue z_getmigrationstatus(const UniValue& params, bool fHelp) {
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
-    if (fHelp || params.size() != 0)
+    if (fHelp || params.size() > 1)
         throw runtime_error(
-            "z_getmigrationstatus\n"
+            "z_getmigrationstatus ( asOfHeight )\n"
             "Returns information about the status of the Sprout to Sapling migration.\n"
             "Note: A transaction is defined as finalized if it has at least ten confirmations.\n"
             "Also, it is possible that manually created transactions involving this wallet\n"
             "will be included in the result.\n"
+            "\nArguments:\n"
+            "1. asOfHeight       (numeric, optional) Execute the query as if it were run when\n"
+            "                    the blockchain was at the height specified by this argument.\n"
+            "                    The default is to use the entire blockchain that the node is\n"
+            "                    aware of.\n"
```

Comment:
Document what happens if `asOfHeight` is greater than the height of the last known block.